src/launcher/parser/nfs/mod.rs

```diff
@@ -64,7 +64,7 @@ fn get_action<D>(action_str: &str, version: f32, decoder: &mut D) -> Result<Box<
             _   => return version_err,
         },
         "create-file" => match version {
-            1.0 => unimplemented!(),
+            1.0 => Box::new(try!(parse_result!(create_file_v1_0::CreateFile::decode(decoder), ""))),
```

Comment:
this file change is strangely not showing up in reviewable tool for me - just quickly mentioning it here:
This will need to contain a parse of `data` field before going through - the rest of them are updated in master - so if you could just draw reference from them and tweak this bit.